src/test/ensemble_config_change_stress.cpp

```diff
@@ -1293,17 +1477,32 @@ class StressPipelineConfigChanges : public TestWithTempDir {
             if (!inputNameCorrect) {
                 return false;
             }
-            bool inputTypeCorrect{inputs.at(pipelineInputName.c_str()).dtype() == tensorflow::DT_FLOAT};
-            EXPECT_TRUE(inputTypeCorrect);
-            if (!inputTypeCorrect) {
-                return false;
-            }
-            bool inputShapeCorrect{isShapeTheSame(
-                inputs.at(pipelineInputName.c_str()).tensor_shape(),
-                {1, 10})};
-            EXPECT_TRUE(inputShapeCorrect);
-            if (!inputShapeCorrect) {
-                return false;
+            if (!isMediapipe) {
```

Comment:
Here we mix mediapipe scheduler stress testing inside "ensemble" unit test cpp file. It might be already confusing that we mix, so I'd suggest to at least add comment what !isMediapipe means - `// means we stress DAG pipeline`